nav_msgs/Path generated_path
---
std_msgs/Bool ok